releasenotes/notes/1.2/deprecate_assemble-67486b4d0a8d4f96.yaml

```diff
@@ -1,7 +1,12 @@
 ---
 deprecations_providers:
   - |
-    The `Qobj` structure and related classes are now deprecated, they were introduced as part of the `BackendV1` workflow and are no longer necessary for interacting with  :class:`.BackendV2` backends. Remote backend interaction should be done via `QPY` or `OpenQASM` instead.
+    The ``Qobj`` structure and related classes are now deprecated, they were introduced as part of 
```

Comment:
Is there a reason to not link to the class here?

Reply:
I'll add a link to the BackendV1 👍🏻 for Qobj I now just linked to the `qiskit.qobj` module, is that fine?